.prettierignore

```diff
@@ -0,0 +1,9 @@
+# Ignore artifacts:
+.github
+.husky
+**/CHANGELOG.md
+bin
+coverage
+node_modules
+vendor
+api.md
```

Comment:
There should be another return here.

https://prettier.io/docs/en/options.html#end-of-line